test/getGrist.ts

```diff
@@ -63,10 +63,10 @@ export class GristTestServer {
     await this.stop();
     const {gristContainerName, gristImage, gristPort, contentPort} = serverSettings;
     const cmd = `docker run -d --rm --name ${gristContainerName}` +
-      ` --network="host"` +
+      ' --add-host=host.docker.internal:host-gateway' +
```

Comment:
What role does host-gateway play?

Reply:
I don't have a Linux machine to test with (though I will go ahead and remove the flag and re-run the CI to check), but I believe this is required for `host.docker.internal` to work on Linux.

Reply:
What do you think about checking the OS before building the docker command, and only applying these changes if it's not Linux? If I understand the situation right on Windows, it has the same limitation as macOS.